internal/resources/resource_host.go

```diff
@@ -140,7 +143,25 @@ func hostSchema() map[string]*schema.Schema {
 		hConnections: {
 			Type:        schema.TypeMap,
 			Computed:    true,
-			Description: "A map of network connections and assigned IP addreses, eg {'Private':'10.83.0.17'}.",
+			Description: "A map of network connection name to assigned IP addrese, eg {'Private':'10.83.0.17'}.",
+		},
+		hConnectionsSubnet: {
+			Type:        schema.TypeMap,
+			Computed:    true,
+			Description: "A map of network connection name to subnet IP address.",
+		},
+		hConnectionsGateway: {
+			Type:        schema.TypeMap,
+			Computed:    true,
+			Description: "A map of network connection name to gateway IP address.",
+		},
+		hConnectionsVLAN: {
```

Comment:
Per API doc, subnet/gateway/vlan are sub-fields of connections. Since it's an existing file and not sure it was discussed to handle as first-level fields in TF.
<img width="579" alt="image" src="https://user-images.githubusercontent.com/65302544/194172351-cfa3e12c-eba3-4227-abf8-02a9d3ee1bb4.png">

Other options could be,
1. Work with LH to expect this change in the upcoming release
2. Deprecate this field and add a new field to present connection details in the same hierarchical fashion.

Reply:
I discusses this with Mark, and we thought it would be simpler to represent this as first-level fields, along with not breaking anyone.  He thought it would be challenging to make sure everyone was notified of the change.  Is it a goal to keep the terraform representation close/the same as the metal client api?